buildlib/az-distro-release.yml

```diff
@@ -62,20 +59,31 @@ jobs:
 
       - bash: |
           set -eEx
+
+          # Build
           ./autogen.sh
           ./contrib/configure-release --with-cuda --with-java=no
           make dist
           tarball=$(echo ucx*.tar.gz)
           tar -xzvf ${tarball}              # extract the sources in a subdirectory
           cd $(tar tf ${tarball} | head -1) # go to extracted tarball directory
           echo 10 > debian/compat   # https://www.debian.org/doc/manuals/maint-guide/dother.en.htmdpl#compat
-          dpkg-buildpackage -us -uc -Pcuda
+          dpkg-buildpackage -d -us -uc -Pcuda
           cd ..                             # Move back to the working directory
-          find . -name '*.deb'
-          VER="${POSTFIX#ucx-}"   # Remove 'ucx' prefix from the POSTFIX string
-          # Rename DEB files
+
+          # Rename DEB files          
+          VER="${POSTFIX#ucx-}"     # Remove 'ucx-' prefix from the POSTFIX string
           find . -name "ucx*.deb" -exec bash -c 'mv "$1" "${1%%_*}-'"${VER}"'.deb"' _ {} \;
+          find . -name '*.deb'      # Show new names
+
+          # Remove superfluous dependency
```

Comment:
Please specify which exact deps you're avoiding

Reply:
Removing dependency on 'libnvidia-compute-<nv_driver_ver> | libnvidia-ml1', so a user would not be locked into a specific NVIDIA driver's version. The actual dependencies required are satisfied by default CUDA installation.

Reply:
Agreed upon resolving it for now.